<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fiver</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QStackedWidget" name="operationsStack">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="fillPage">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QGridLayout" name="fillPageLayout">
            <item row="1" column="1">
             <widget class="QPushButton" name="appendVote">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>400</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QSlider" name="voteValue">
              <property name="minimum">
               <number>-4</number>
              </property>
              <property name="maximum">
               <number>4</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="image">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="analysePage">
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <layout class="QGridLayout" name="analysePageLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Термин</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QGroupBox" name="resourceBox">
              <property name="title">
               <string>Ресурсы</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
              <layout class="QGridLayout" name="gridLayout_10">
               <item row="0" column="1">
                <widget class="QPushButton" name="appendResource">
                 <property name="text">
                  <string>Добавить...</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="deleteResource">
                 <property name="text">
                  <string>Удалить</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <spacer name="verticalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="0" column="0" rowspan="3">
                <widget class="QTableView" name="resourceList">
                 <attribute name="horizontalHeaderStretchLastSection">
                  <bool>true</bool>
                 </attribute>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="analyse">
              <property name="text">
               <string>Анализировать</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="vocabularyEditPage">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QGridLayout" name="vocabularyEditLayout">
            <item row="0" column="1">
             <widget class="QPushButton" name="deleteArticle">
              <property name="text">
               <string>Удалить</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="addArticle">
              <property name="text">
               <string>Добавить...</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" rowspan="4">
             <widget class="QTableView" name="articleList">
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <attribute name="horizontalHeaderCascadingSectionResizes">
               <bool>true</bool>
              </attribute>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>150</number>
              </attribute>
              <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
               <bool>true</bool>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderHighlightSections">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderShowSortIndicator" stdset="0">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderStretchLastSection">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="changeVoteBtn">
              <property name="text">
               <string>Изменить оценку...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Инструменты</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>Словарь</string>
     </property>
     <addaction name="vocEditAction"/>
     <addaction name="vocFillAction"/>
    </widget>
    <widget class="QMenu" name="menu_5">
     <property name="title">
      <string>Информационное поле</string>
     </property>
     <addaction name="analyseAction"/>
    </widget>
    <addaction name="menu_4"/>
    <addaction name="menu_5"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="docAction"/>
    <addaction name="separator"/>
    <addaction name="aboutAction"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="optionsAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_6"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="vocEditAction"/>
   <addaction name="vocFillAction"/>
   <addaction name="separator"/>
   <addaction name="analyseAction"/>
   <addaction name="separator"/>
   <addaction name="optionsAction"/>
  </widget>
  <widget class="QStatusBar" name="status"/>
  <action name="exitAction">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="docAction">
   <property name="text">
    <string>Руководство пользователя</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>О программе...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="vocEditAction">
   <property name="text">
    <string>Редактор</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+E</string>
   </property>
  </action>
  <action name="vocFillAction">
   <property name="text">
    <string>Заполнение</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+F</string>
   </property>
  </action>
  <action name="analyseAction">
   <property name="text">
    <string>Анализ</string>
   </property>
   <property name="shortcut">
    <string>Alt+Shift+A</string>
   </property>
  </action>
  <action name="optionsAction">
   <property name="text">
    <string>Настройки...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
